### PR TITLE
Minor fixes in design and readme

### PR DESCRIPTION
**Pull request by resouer**
Refactored the design doc a bit. No significant change expect reorder/rewrite sentences.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ For platform builders, KubeVela serves as a framework that empowers them to crea
 - Slack: [Discuss](https://cloud-native.slack.com/archives/C01BLQ3HTJA)
 - Gitter: [Community](https://gitter.im/oam-dev/community)
 
+> NOTE: KubeVela is still an early project and iterating quickly. It's currently under preview release.
```
Comment (Fei-Guo) [inline README.md:14]: is still an early project -> is still in early stage?

File: docs/design.md [hunk 2]
```diff
@@ -22,58 +18,53 @@ As a platform engineer, I want to build a easy-to-use platform for end users. In
 - Heroku-like (_in terms of both user experience and functionality_). 
   - and I prefer to build my own version with OSS tools, particularly, with Kubernetes (for obvious reason).
 - Easy to build. 
-  - I am too busy to reinvent any wheel, I want to reuse every capability in Kubernetes community as possible,  with minimal effort. Writing some simple CRD and controllers is fine, but please, just the simple ones like copy-paste.
+  - I am too busy to reinvent any wheel, I want to reuse every capability in Kubernetes community as possible,  with minimal effort. Writing some simple CRD and controllers is fine, but please, just the simple ones.
```
Comment (Fei-Guo) [inline docs/design.md:21]: reuse every capability -> reuse as many capabilities

**Review by Fei-Guo — COMMENTED** [review]
A few Nits. I will go through design doc later after check in.

File: docs/design.md (hunk 2 above)
Comment (resouer) [reply docs/design.md]: Done, all fixed!

File: docs/design.md [hunk 3]
```diff
@@ -22,58 +18,53 @@ As a platform engineer, I want to build a easy-to-use platform for end users. In
 - Heroku-like (_in terms of both user experience and functionality_). 
   - and I prefer to build my own version with OSS tools, particularly, with Kubernetes (for obvious reason).
 - Easy to build. 
-  - I am too busy to reinvent any wheel, I want to reuse every capability in Kubernetes community as possible,  with minimal effort. Writing some simple CRD and controllers is fine, but please, just the simple ones like copy-paste.
+  - I am too busy to reinvent any wheel, I want to reuse as many capabilities in Kubernetes community as possible, with minimal effort. Writing some simple CRD and controllers is fine, but please, just the simple ones.
```
Comment (hongchaodeng) [inline docs/design.md:21]:
```suggestion
  - I am too busy to reinvent any wheel, I want to reuse the existing capabilities in Kubernetes community with minimal effort. Writing some simple CRD and controllers is fine, but please, just the simple ones.
```

File: docs/design.md [hunk 4]
```diff
@@ -22,58 +18,53 @@ As a platform engineer, I want to build a easy-to-use platform for end users. In
...
 - Powerful and highly extensible.
   - I don't want to lock users with restricted abstractions and capabilities like traditional PaaS or FaaS/Serverless. I love Kubernetes and what it has enabled. So in terms of capability, I hope my platform is fully open and has unlimited possibilities like Kubernetes itself, rather than another opinionated close system like traditional PaaS.
 
-
 ## Core Principles
 
-In nutshell, the principles for KubeVela project are:
+For developers, KubeVela out-of-the-box provides a PaaS-like experience that enables better productivity (i.e. focus, self-service, on-demand, and fast-feedback).
```
Comment (hongchaodeng) [inline docs/design.md:27]:
```suggestion
For developers, KubeVela provides a out-of-the-box PaaS-ish experience that enables better productivity (i.e. focus, self-service, on-demand, and fast-feedback).
```

File: docs/design.md [hunk 5]
```diff
@@ -22,58 +18,53 @@ As a platform engineer, I want to build a easy-to-use platform for end users. In
...
 
-- For end users, it out-of-the-box provides a fully featured PaaS-like experience, nothing special.
-- For platform builders, it works like a special Kubernetes "distro" or extensible PaaS core that could be used to build something more complex on top of. It allows platform builders to integrate any existing capabilities in Kubernetes ecosystem to end users with minimal effort, or develop a new capability at ease in a standardized and Kubernetes-native approach.
+For platform builders, KubeVela works like a special Kubernetes "distro" or extensible PaaS core that could be used to build something more complex on top of. It enables platform builders to integrate any existing capabilities in Kubernetes ecosystem with minimal effort, or develop a new capability at ease in a standardized and Kubernetes-native approach.
```
Comment (hongchaodeng) [inline docs/design.md:29]:
```suggestion
For platform builders, KubeVela works like a special Kubernetes "distro" or extensible PaaS core that could be used to build something more complex on top of. It enables platform builders to integrate existing capabilities in Kubernetes ecosystem with minimal effort, and/or develop a new capability at ease in a standard, Kubernetes-native approach.
```

File: docs/design.md [hunk 6]
```diff
@@ -22,58 +18,53 @@ As a platform engineer, I want to build a easy-to-use platform for end users. In
...
 
 ## Design Details
 
 ### 1. Application Centric
 
-The API and interfaces of KubeVela intends to make users think in terms of application, not containers or infrastructure. 
+Keywords: _focus_, _self-service_.
 
-Lacking application context impacts the user experience and significantly raised the bar to adopt cloud native stack. We believe "application" is the natural mindset of developers and it's the core concept an application platform should expose.
+KubeVela intends to ensure developers only think in terms of _application_, not container or infrastructure. Lacking application context impacts the developer experience and significantly raised the bar to adopt cloud native technology. We believe _application_ is the natural mindset of developers and it's the core concept KubeVela should expose.
```
Comment (hongchaodeng) [inline docs/design.md:37]:
```suggestion
KubeVela intends to ensure developers only think in the scope of _application_, not container or infrastructure. Lacking application context impacts the developer experience and raises the bar to adopt cloud native technology. We believe _application_ is the natural mindset for developers and it's the core concept KubeVela should expose.
```

File: docs/design.md [hunk 7]
```diff
@@ -22,58 +18,53 @@ As a platform engineer, I want to build a easy-to-use platform for end users. In
...
 
 ![alt](../resources/app-centric.png)
 
-KubeVela intends to let developers push code, define application in developer facing primitives, and make daily operations as configurations of the application.  
-
-Thus, KubeVela choose to:
-1. Introduce "application" as first class citizen and main API.
+In KubeVela, developers will only push code, deploy application in developer facing primitives, and claim operations policies as part of the application definition. Thus, KubeVela choose to:
```
Comment (hongchaodeng) [inline docs/design.md:41]:
```suggestion
In KubeVela, it is only push code, deploy application in developer facing primitives, and claim operations policies that developers would do. Thus, KubeVela choose to:
```

File: docs/design.md [hunk 8]
```diff
@@ -22,58 +18,53 @@ As a platform engineer, I want to build a easy-to-use platform for end users. In
...
 
 ### 2. Capability Oriented Architecture
 
-To enable platform builders use KubeVela to create their own application platforms in an easy and Kubernetes native approach, KubeVela intends to make its every capability a standalone "plug-in".
+Keywords: _on-demand_.
+
+KubeVela intends to make its every capability a standalone "plug-in" in terms of implementation and configuration. This enables the extensibility of the platform, and gives platform builders full freedom to extend KubeVela into anything else they want (e.g. database PaaS, AI PaaS, etc.).
```
Comment (hongchaodeng) [inline docs/design.md:53]:
```suggestion
Every capability in KubeVela is designed as a standalone "plug-in" in terms of implementation and configuration. This enables the extensibility of the platform, and gives platform builders full freedom to extend KubeVela into anything else they want (e.g. database PaaS, AI PaaS, etc.).
```

**Comment by resouer** [comment]
@hongchaodeng All comments fixed. After merge, Fei Guo will take a second round of fix.

**Review by wonderflow — APPROVED** [review]

**Pull request merged**